spras/omicsintegrator2.py

```diff
@@ -125,7 +125,8 @@ def run(edges=None, prizes=None, output_file=None, w=None, b=None, g=None, noise
                             container_suffix,
                             command,
                             volumes,
-                            work_dir)
+                            work_dir,
+                            network_disabled=True)
```

Comment:
The `network_disabled` parameter should be added into the config.yaml file and the run function arguments as a parameter a user can set.

Reply:
Does this parameter apply to all algorithms or only oi2?

I'm assuming all, but for the scope of this PR I would keep it as an oi2 only parameter that can be used and set in the config file. If needed, a new PR can be created to enable this for all docker containers.